src/schema.ts

```diff
@@ -43,15 +46,30 @@ export class Schema {
         }
       } else if (primitiveType === PrimitiveType.DATE) {
         type = Type.TEMPORAL;
+        // need to get correct min/max of date data
```

Comment:
Datalib's `inferAll` method works correctly, but datalib's `summary` method doesn't get the type correct. It considers the length of the date string when it determines the stats, so we have to manually figure out the min/max date.